internal/pkg/config/testdata/config/full.yaml

```diff
@@ -18,10 +18,12 @@ trust_zones:
       extra_helm_values:
         global:
             spire:
-                namespaces:
-                    create: true
+                caSubject:
+                    commonName: cn.example.com
+                    organization: acme-org
```

Comment:
Should this match the SPIRE CASubject (i.e. `"Cofide") as defined below?

Reply:
The tz1 fixture has a helm value override in it's config that modifies the CA subject.